crates/node/src/types/transaction.rs

```diff
@@ -250,6 +250,20 @@ impl Payload {
             }
         }
     }
+
+    pub fn get_execution_output(self) -> Option<(Vec<TxFile<Output>>, Vec<u8>)> {
+        match self {
+            Payload::Proof {
+                proof: data, files, ..
+            }
+            | Payload::Verification {
+                verification: data,
+                files,
+                ..
+            } => Some((files, data)),
```

Comment:
Should the return fields be maybe `(data, files)` when that's the order throughout the code base (including gRPC's `TaskResult`)?